addons/website_sale_product_configurator/static/src/js/website_sale_options.js

```diff
@@ -113,6 +114,8 @@ publicWidget.registry.WebsiteSale.include({
                         const $quantity = $(".my_cart_quantity");
                         $quantity.parent().parent().removeClass('d-none');
                         $quantity.text(quantity).hide().fadeIn(600);
+                    }).then(()=>{
+                        this._getCombinationInfo($("#add_to_cart_wrap"));
```

Comment:
Wouldn't it be easier to simply refresh the page instead of modifying the `_getCombinationInfo` API (which seems big for stable bugfix) ?

Reply:
we could also just give `$.Event('click', {target: $("#add_to_cart_wrap")})` and not change the signature